code/modules/antagonists/traitor/datum_traitor.dm

```diff
@@ -242,14 +242,18 @@
 		return
 	var/mob/traitor_mob=owner.current
 
-	to_chat(traitor_mob, "<U><B>The Syndicate provided you with the following information on how to identify their agents:</B></U>")
-	to_chat(traitor_mob, "<B>Code Phrase</B>: <span class='danger'>[GLOB.syndicate_code_phrase]</span>")
-	to_chat(traitor_mob, "<B>Code Response</B>: <span class='danger'>[GLOB.syndicate_code_response]</span>")
+	var/phrases = jointext(GLOB.syndicate_code_phrase, ", ")
+	var/responses = jointext(GLOB.syndicate_code_response, ", ")
 
-	antag_memory += "<b>Code Phrase</b>: [GLOB.syndicate_code_phrase]<br>"
-	antag_memory += "<b>Code Response</b>: [GLOB.syndicate_code_response]<br>"
+	to_chat(traitor_mob, "<U><B>The Syndicate have provided you with the following codewords to identify fellow agents:</B></U>")
+	to_chat(traitor_mob, "<B>Code Phrase</B>: <font color=blue>[phrases]</font>")
+	to_chat(traitor_mob, "<B>Code Response</B>: <font color=red>[responses]</font>")
 
-	to_chat(traitor_mob, "Use the code words in the order provided, during regular conversation, to identify other agents. Proceed with caution, however, as everyone is a potential foe.")
+	antag_memory += "<b>Code Phrase</b>: <font color=blue>[phrases]</font><br>"
+	antag_memory += "<b>Code Response</b>: <font color=red>[responses]</font><br>"
+
+	to_chat(traitor_mob, "Use the codewords during regular conversation to identify other agents. Proceed with caution, however, as everyone is a potential foe.")
+	to_chat(traitor_mob, "<font size=3 color=red>You memorize the codewords, allowing you to recognise them when heard.</font>")
```

Comment:
All these \<font\> tags should really be \<span\> tags instead

Reply:
For doing "color=", everywhere else in the code all use the font tag. Not entirely sure myself but it seems the standard way to do it?

Reply:
Font tags are used to highlight something in the middle of a message, span class tags for the entirety of the message (and are basically a shorthand for specific size/color options), AFAIK.

Reply:
I can change to <span style="color="...""> if need be, but it's messy and needing escaped speech marks. In the greet text for traitors, the "You are the Traitor." text is also done with the font tag, as there is no set class for that specific colour/size.

Reply:
If it needs a new class, just add one in the goonchat css. Hardcoded font styles end up being a pain if someone wants to change it later down the line. You can also do spicy stuff like animations with span css much easier.

Reply:
Fair enough, making new classes for them, I'll move over to <span>.